src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,8 +5,7 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
The class 'Truck' should not start with an empty line. Please remove the empty line at the beginning of the class implementation.